<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="MBP-Kubanychbek.attlocal.net" name="api.RunnerClass" tests="1" failures="1" timestamp="05 Nov 2018 02:59:04 GMT" time="1.684" errors="0">
  <testcase name="runScenario" time="1.684" classname="api.RunnerClass">
    <failure type="java.lang.AssertionError" message="expected:&lt;200&gt; but was:&lt;422&gt;">
      <![CDATA[java.lang.AssertionError: expected:<200> but was:<422>
at org.junit.Assert.fail(Assert.java:88)
at org.junit.Assert.failNotEquals(Assert.java:834)
at org.junit.Assert.assertEquals(Assert.java:645)
at org.junit.Assert.assertEquals(Assert.java:631)
at implementedSteps.Test1.user_should_get_as_a_status_code(Test1.java:35)
at âœ½.user should get 200 as a status code(./src/test/java/features/gitHubSearch.feature:7)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- api.RunnerClass -->
